### PR TITLE
Update community.yml

### LINKED ISSUE #72: Update Community.yml
I will update the issue-message, Pr-Message and footer with new ones in Community.yml

### PR DESCRIPTION
**Pull request by muthuannamalai12**
closes #72 

I have updated the issue-message, Pr-Message, and footer with new ones in Community.yml

### CODE REVIEW

File: .github/workflows/community.yml [hunk 1]
```diff
@@ -17,17 +17,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: ./src/welcome
+      - uses: EddieJaoudeCommunity/gh-action-community/src/welcome@main
```
Comment (eddiejaoude) [inline .github/workflows/community.yml:20]: This should be `EddieHubCommunity`

File: .github/workflows/community.yml [hunk 2]
```diff
@@ -17,17 +17,17 @@ jobs:
...
+          issue-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          pr-message: '<h1>It''s great having you contribute to this project</h1> Welcome to the community :nerd_face:'
+          footer: 'If you would like to continue contributing to open source and would like to do it with an awesome inclusive community, you should join our <a href="http://discord.eddiehub.org">Discord</a> chat and our <a href="http://github.eddiehub.org">GitHub Organisation</a> - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'
 
   statistics:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: ./src/statistics
+      - uses: EddieJaoudeCommunity/gh-action-community/src/statistics@main
```
Comment (eddiejaoude) [inline .github/workflows/community.yml:31]: Same here `EddieHubCommunity`

**Comment by eddiejaoude** [comment]
Looks good 👍 , a couple of comments

File: .github/workflows/community.yml [hunk 3]
```diff
@@ -17,17 +17,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: ./src/welcome
+      - uses: EddieHubCommunity/gh-action-community/src/welcome@main
```
Comment (AllanRegush) [inline .github/workflows/community.yml:20]: Shouldn't we be using the latest release not main?

Comment (eddiejaoude) [reply .github/workflows/community.yml:20]: Release would be better 👍 , but I think it was using `main` before. This can be updated here or in another PR

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**